persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/interceptors/ProxyProtocolQueryInterceptor.java

```diff
@@ -117,6 +114,7 @@ private ProxyProtocolQueryInterceptor(
       QueryInterceptor wrapped) {
     this.resolver = resolver;
     this.proxyDnsName = proxyDnsName;
+    this.internalProxyDnsName = "internal-" + proxyDnsName;
```

Comment:
We should pull this from a system property as well rather than requiring a naming convention.

Reply:
I was taking the path of least resistance because a new property will have more impact in downstream projects.

But actually I can have the best of both worlds by using the convention if the new property is absent. Will do.